### PR TITLE
Base layer information

### PR DESCRIPTION
**Pull request by EnBakir**
Displays information about the undermost layer.

### CODE REVIEW

File: test/modules/map/components/geoResourceInfo/GeoResourceInfo.test.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+				layers: {
+					active: [layer]
+				}
+			};
+			const geoServiceMock = spyOn(geoResourceServiceMock, 'byId').withArgs(layer.id).and.returnValue(null);
+			const element = await setup(state);			
+
+			expect(element.shadowRoot.querySelector('.geo-rsrc-info')).toBeFalsy();
+			expect(geoServiceMock).toHaveBeenCalledOnceWith(layer.id);
+		});
```
Comment (taulinger) [inline test/modules/map/components/geoResourceInfo/GeoResourceInfo.test.js:66]: we also should have a test on changing the layer on index=0 after initialisation
```
it('updates GeoResourceInfo component') ...
```

**Review by taulinger — COMMENTED** [review]
Fine!

**Review by thiloSchlemmer — APPROVED** [review]

File: src/modules/footer/i18n/baseLayerInfo.provider.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+export const provide = (lang) => {
+	switch (lang) {
+
+		case 'en':
+			return {
+				//the first part of the snake_case key should be the name of the related module
+				map_baseLayerInfo_label: 'Base Layer'
```
Comment (taulinger) [inline src/modules/footer/i18n/baseLayerInfo.provider.js:7]: Let's call it 'Basemap'

**Review by taulinger — COMMENTED** [review]
Well done! 
@alsturm Could you take a look at the design?

**Review by taulinger — APPROVED** [review]

**Comment by taulinger** [comment]
We have to wait with the merge until @alsturm did his review

**Review by taulinger — APPROVED** [review]

**Review by alsturm — APPROVED** [review]

**Review by alsturm — APPROVED** [review]
nice! just removed theme toggle...

**Pull request merged**